cli/src/maintenance.rs

```diff
@@ -403,6 +433,53 @@ impl SolidoState {
         Some((instruction, task))
     }
 
+    /// If there is a validator being deactivated, try to unstake its funds.
+    // TODO(#386): Solana could fail the transaction to unstake, in that case,
+    // it should be retried.
+    pub fn try_unstake_from_inactive(&self) -> Option<(Instruction, MaintenanceOutput)> {
+        for validator in &self.solido.validators.entries {
+            // Validator is inactive, try to unstake from it.
+            if !validator.entry.active {
```

Comment:
This is a bit a matter of preference, but we can write

```rust
// We are only interested in unstaking from inactive validators.
if validator.entry.active {
    continue
}
```
and that avoids some rightward drift because the remainder of the function doesn’t have to be inside the `if`.